### PR TITLE
v2.7.1 - Fix bug can't place order due to issue during load order by entity_id

### PR DESCRIPTION
**Pull request by Xaxxis**
Fix Bug:
- Can't place order due to can't load order when increment_id is not equal with entity_id
- Fix get order by increment id function

### CODE REVIEW

File: Controller/Payment/Notification.php [hunk 1]
```diff
@@ -32,7 +32,9 @@ public function execute()
 
             $this->getResponse()->setBody('ok');
             foreach ($incrementIds as $id) {
-                $order  = $this->paymentOrderRepository->getOrderByIncrementId($id);
+                $entityId = $this->paymentOrderRepository->getOrderByIncrementId($id)->getEntityId();
+                $order  = $this->paymentOrderRepository->getOrderByEntityId($entityId);
+                $order->getEntityId();
```
Comment (rizdaprasetya) [inline Controller/Payment/Notification.php]: Seems like the return value of `$order->getEntityId();` is not stored in any of the variable. Is this typo or expected to call that function without needing the return value?

**Review by rizdaprasetya — COMMENTED** [review]
All looks good, except one item. If that's addressed, then IMO good to merge & release @Xaxxis thanks.

File: Controller/Payment/Notification.php (hunk 1 above)
Comment (Xaxxis) [reply Controller/Payment/Notification.php]: Unused code @rizdaprasetya, already removed. Thanks

**Review by rizdaprasetya — APPROVED** [review]
Cool thanks, looks good to me, please go ahead @Xaxxis

**Comment by Xaxxis** [comment]
Released on Magento marketplace v2.7.1

**Pull request merged**